Barotrauma/BarotraumaShared/SharedSource/Events/Missions/AbandonedOutpostMission.cs

```diff
@@ -335,18 +340,23 @@ protected override void UpdateMissionSpecific(float deltaTime)
                         State = 1;
                     }                    
                     break;
-#if SERVER
                 case 1:
-                    if (!(GameMain.GameSession.GameMode is CampaignMode) && GameMain.Server != null)
+                    if (GameMain.GameSession.GameMode is not CampaignMode)
                     {
                         if (!Submarine.MainSub.AtStartExit || (wasDocked && !Submarine.MainSub.DockedTo.Contains(Level.Loaded.StartOutpost)))
                         {
-                            GameMain.Server.EndGame();
+#if SERVER
+                            GameMain.Server?.EndGame();
+#else
+                            if (GameMain.GameSession.GameMode is { IsSinglePlayer: true } && GameMain.GameSession.CrewManager is { IsSinglePlayer: true })
```

Comment:
Same here, why does `IsSinglePlayer: true` need to be checked on both of these?

Reply:
Fixed; see previous comment.